File: c/1173.c
#include <stdio.h>

int main ()
{

  int n[10], i, x, k;

  scanf("%d", &x);

  for (k = 0, i = x; k < 10; i *= 2, k++)
  {

    n[k] = i;

  }

  for (i = 0; i < 10; i++)
  {

    printf("N[%d] = %d\n", i, n[i]);

  }
}

File: go/1173.go
package main

import "fmt"

func main() {
	var arr []int
	var x int

	fmt.Scanf("%d", &x)
	for i := 0; i < 10; i++ {

		arr = append(arr, x)
		x *= 2
	}

	for i, v := range arr {
		fmt.Printf("N[%d] = %d\n", i, v)
	}
}
